homeassistant/components/binary_sensor/ihc.py

```diff
@@ -0,0 +1,183 @@
+"""IHC binary sensor platform."""
```

Comment:
Use our standard format for the module docstrings:
https://home-assistant.io/developers/development_guidelines/#file-headers

Reply:
Done